docs/src/extend/plugin-migration-flat-config.md

```diff
@@ -185,20 +187,26 @@ module.exports = plugin;
 Your users can then use this exported config like this:
 
 ```js
+import { defineConfig } from "eslint/config";
 import example from "eslint-plugin-example";
 
-export default [
+export default defineConfig([
 
     // use recommended config
     example.configs.recommended,
 
     // and provide your own overrides
```

Comment:
```suggestion
    // use recommended config and provide your own overrides
```

Adding `example.configs.recommended` separately seems unnecessary in this example as it's added in `extends` below?